exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -118,22 +120,48 @@ def get_resource_data(
     return _get_resource_data(sdk_resource_scope_data, resource_class, name)
 
 
-def _load_credential_from_file(filepath) -> ChannelCredentials:
+def _get_file_content(file_path: str) -> bytes:
```

Comment:
Tiny nit, but can we rename this to perhaps `_read_file`?

Reply:
Fixed, tyvm!